### ISSUE TITLE
SMTP server doesn't support "AUTH <mechanism> <initial data>" for of authentication

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11086]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1552|
|Type|defect|
|Created|2006-03-06 22:22:17Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1552 1552
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1141683737000000 1141683737000000
changetime__1157665488000000 1157665488000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

Read for review in smtp-initial-response-1552

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to @exarkun|
|-|-|

That challenger stuff is terrible. And it's in imap4. Dangit.

merge it plz

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [18075]) Merge smtp-initial-response-1552

Author: exarkun
Reviewer: radix
Fixes [#11086](https://github.com/twisted/twisted/issues/11086)

This adds support for initial-response arguments to the ESMTP AUTH command (eg,
AUTH PLAIN ZXhhcmt1bg==).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
